source/aggregation-pipeline-builder.txt

```diff
@@ -89,69 +89,226 @@ Limitations
-
-   .. step:: Enter valid aggregation syntax in the text editor. 
-      The following example pipeline limits the query results to ``4`` 
-      documents.
-
-      .. code-block:: javascript
-
-         [ { "$limit" : 4 } ]
-
-      .. figure:: /images/compass/agg-builder-textview-edit.png
-         :alt: Edit the text aggregation pipeline
-
-   .. step:: **(Optional)** If preview mode is enabled, you can 
-      click :guilabel:`Output Options` in the Pipeline Output pane to 
-      expand all nested fields and documents within the 
-      preview results.
-
-      .. figure:: /images/compass/agg-builder-textview-expand-pipeline-preview.png
-         :alt: Pipeline Output preview
-
-   .. step:: Click :guilabel:`Run` at the top right of the pipeline builder. 
-      |compass-short| returns your results in the document view.
-
-      .. figure:: /images/compass/agg-builder-textview-agg-results.png
-         :alt: Results after Running Aggregation
-
-   .. step:: **(Optional)** Click the :guilabel:`Output Options` dropdown 
-      and select :guilabel:`Expand all fields` to expand all nested 
-      fields and documents within the query results.
-
-      .. figure:: /images/compass/agg-builder-textview-expand-results.png
-         :alt: Expand the aggregation pipeline results
+To see how to create an aggregation pipeline, select the tab corresponding to 
+your view mode of choice:
+
+.. tabs:: 
+
+   .. tab:: Stage View Mode 
+      :tabid: stage 
+
+      .. procedure:: 
+         :style: normal
+
+         .. step:: In the aggregation pipeline pane, ensure the 
+            :guilabel:`{} Stages` toggle switch is selected.
+
+            .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
+               :alt: Toggle on stage view mode 
+
+         .. step:: At the bottom of the aggregation pipeline pane, select the 
+            :guilabel:`+ Add Stage` button to open a new aggregation stage 
+            modal.
+            
+         .. step:: At the lower-left corner of the aggregation pipeline pane, 
+            click the :guilabel:`Select...` dropdown and select the 
+            :manual:`aggregation pipeline stage 
+            </reference/operator/aggregation-pipeline/>` to use for the first 
+            stage of the pipeline. 
+
+         .. step:: Fill in your selected pipeline stage. 
+            
+            If you enable :guilabel:`Preview`, preview documents update 
+            automatically to the right of the stage to reflect the results of 
+            your pipeline as it progresses. You can adjust the width of the 
+            pipeline stage by dragging its border to the right.
+            
+            .. figure:: /images/compass/agg-builder-match-stage.png 
+               :alt: Aggregation Builder match stage example
+         
+         .. step:: Click :guilabel:`Add Stage` to add additional aggregation 
+            stages below your last aggregation stage. Repeat steps 1 and 2 for 
```

Comment:
[question]

Does "last" here mean "last in sequential order", or "most recently added" ?

Reply:
This refers to "most recently added" will update this!

Reply:
Update: Previously in stage view mode, you could only add stages after your last sequential stage. In the new version, it refers to the most recently added stage. I'll add a note to make this change in the next ticket in this project regarding changes to the Stage View UI.